webrtc/interfaces.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html>
+  <head>
+    <title>"WebRTC IDL Tests</title>
```

Comment:
I see. Good to know some examples. Will make use of the simpler styles in new PRs then.